### PR TITLE
Edit source settings

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

**Comment by tpluscode** [comment]
@martinmaillard are we good to merge #133 and rebase here to continue on the API end?

**Comment by martinmaillard** [comment]
Missing:
- [ ] API endpoint to update source

It should update the source dialect and re-parse the CSV file.

File: apis/core/lib/domain/csv-mapping/update.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  store = resourceStore(),
+  fileStorage = s3,
+}: UpdateCsvSourceCommand): Promise<GraphPointer> {
+  const changed = RdfResource.factory.createEntity<CsvSource>(resource)
+  const csvSource = await store.getResource<CsvSource>(resource.term)
+  if (!csvSource) {
+    throw new NotFoundError(resource)
+  }
+
+  csvSource.name = changed.name
```
Comment (tpluscode) [inline apis/core/lib/domain/csv-mapping/update.ts:33]: Now that you removed the name from the shape, we'd need to remove this assignment too, lest a request clears the name

Or we could bring back the property shape with `dash:hidden`

**Review by tpluscode — APPROVED** [review]

File: apis/core/lib/domain/csv-mapping/update.ts (hunk 1 above)
Comment (martinmaillard) [reply apis/core/lib/domain/csv-mapping/update.ts:33]: Ok I'll re-add the property with `dash:hidden`

**Review by lucafurrer — APPROVED** [review]

**Pull request merged**